sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/FileShareTestBase.java

```diff
@@ -672,6 +672,10 @@ protected static boolean olderThan20230103ServiceVersion() {
         return olderThan(ShareServiceVersion.V2023_01_03);
     }
 
+    protected static boolean olderThan20240204ServiceVersion() {
```

Comment:
Couldn't we just call the `olderThan(ShareServiceVersion targetVersion)` method below instead of adding this version-specific method?

Reply:
@seanmcc-msft This is a pattern we were following for all tests requiring service versions. If you see here:
https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/FileShareTestBase.java#L635

These are all existing methods that we use for the tests when using the **@DisabledIf** tag, like seen here:
https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/FileApiTests.java#L249

I believe the reason why we use new methods instead of calling the existing olderThan(ShareServiceVersion) is because the **@DisabledIf** tag uses string literals for method references.